src/components/BaseComponents/BaseHeader.tsx

```diff
@@ -57,22 +67,24 @@ function BaseHeader(props: HeaderProps) {
     menu: getIcon(history.goBack, <MenuIcon />), //replace history.goBack with correct functions
     backNav: getIcon(history.goBack, <ArrowBackIosIcon />),
     edit: getIcon(navigateToEdit, <CreateIcon />),
-    user: getIcon(history.goBack, <AccountCircleIcon />),
+    user: getIcon(history.goBack, <AccountCircleIcon className={classes.grey} />),
```

Comment:
It's not really a big deal but imo it'd be cleaner to expose these elements themselves so that a developer can choose the elements to pass down rather than a string placeholder that ends up mapping to that element

ex: `<BaseScreen leftIcon={Icons.BACK_NAV} title={customer.name} rightIcon={Icons.EDIT} match={match}>` instead of `<BaseScreen leftIcon="backNav" title={customer.name} rightIcon="edit" match={match}>`.

Not a big deal though right now so feel free to ignore or put as TODO if you think it should be done that way.